issuebot

```diff
@@ -131,6 +164,11 @@ lilac can't parse out the relevant package names, please handle manually.''')
     with file_lock(LILAC_LOCK):
       subprocess.check_output(['git', 'pull'], cwd=REPO)
 
+      for name in os.listdir(REPO):
+        if not os.path.isdir(os.path.join(REPO, name)):
+            continue
+        remove_repo_depends(name, packages)
```

Comment:
We'd set `changed = True` if we've made changes, or they won't be committed. Also we need to give different messages when the package itself has been removed but we are updating lilac.yaml.

Reply:
> Also we need to give different messages when the package itself has been removed but we are updating lilac.yaml.

I'm too lazy to add more if/else. How about changing the commit message from

```
in official repos, removing.
```
to
```
in official repos, removing packages and repo_depends entries if any.
```

Reply:
It's fine.

Reply:
Fixed in the latest commit.